### DISCUSSION TITLE
Facebook strips away query params arrays

### DISCUSSION THREAD

**Discussion by madsh93** [vuejs/router#1815] [Help and Questions] [answered]
I'm not really sure what to do here. But it a bit of an issue for me and my site that is heavily shared on Facebook:

If you send a Facebook Messenger message containing an array in the query params e.g. this link, which is how I understand Vue router handles arrays.

https://www.domain.com/demo?equipment_ids=26&equipment_ids=28

Facebook will strip away the first equipment_ids:

![CleanShot 2023-04-24 at 13 24 52](https://user-images.githubusercontent.com/8860723/233983056-4348007a-3d94-49de-b647-f8d6b2cd4134.png)

Any idea on what is going on?

**Comment by posva** [comment, marked as answer]
I think I answered this on Discord but to add it here too: This seems like a misalignement from Messenger as it's a native feature of URLSearchParams:

```js
p = new URLSearchParams()
p.append('a', 1)
p.append('a', 2)
p.append('a', 3)
p.toString()
'a=1&a=2&a=3'
```

It would be nice if you could send a ticket to Facebook Support as it would be important for them to fix it.

**Reply by madsh93** [reply]
Thanks @posva 

I've contacted Facebook but I'm not really getting through and they won't a knowledge they have a bug.

**Reply by posva** [reply]
It will probably take time. Hopefully not too much 🤞
